testing/unittests/jlibtests.cpp

```diff
@@ -231,6 +231,11 @@ class JlibTraceTest : public CppUnit::TestFixture
             OwnedSpanScope serverSpan = queryTraceManager().createServerSpan("declaredSpanStartTime", emptyMockHTTPHeaders, &declaredSpanStartTime);
             //{ "type": "span", "name": "declaredSpanStartTime", "trace_id": "0a2eff24e1996540056745aaeb2f5824", "span_id": "46d0faf8b4da893e",
             //"start": 1702672311203213259, "duration": 125311051 }
+
+            SpanTimeStamp clientSpanTimeStamp;
```

Comment:
were you able to confirm the start time was adjusted as expected?
We're not able to automatically validate these values (yet) but the log output should match your expected start time (we might need to printout clientSpanTimeStamp.

Reply:
yes, I added the comments for the jlog Span output